README.md

```diff
@@ -29,7 +29,7 @@ To use the module in Terraform add, the segment below to your Terraform configur
 ```
 module "cloudwatch-log-group-x-papertrail" {
     source = "github.com/vaana-tech/cloudwatch-logs-to-papertrail//tf_module?ref=GIT_TAG_TO_USE"
-    monitor_log_group_name = "/aws/lambda/my-lambda-function"
+    monitor_log_group_name = ["/aws/lambda/my-lambda-function", "/aws/some/other/log/group/name"]
```

Comment:
Should be `monitor_log_group_names` to match the terraform declaration.

Reply:
Done